code/src/styles/foundation/_variables.scss

```diff
@@ -115,6 +115,7 @@ $spacing-scale-factor: 0.25rem;
 =============================================*/
 
 $screen-height: 100svh;
+$header-height: 112px;
```

Comment:
I guess it shouldn't be here in general variables in our design system variables
I think we can use value for `height` localy for `scss` it will not a problem to define `height` there

Reply:
Thanks, it makes sense. I moved `header-height` variable to the file where it is used. I still think it is good to store that value inside of a `scss` variable because it is used in `calc` function and without name it is very unclear what we store in that variable.